daffodil-runtime1/src/main/scala/org/apache/daffodil/infoset/ScalaXMLInfosetInputter.scala

```diff
@@ -81,6 +82,21 @@ class ScalaXMLInfosetInputter(rootNode: Node)
         }
         val text = child match {
           case t: Text => t.data
+          //
+          // Note: may be a bug in scala.xml library, but if you do this:
```

Comment:
Submit this to the scala peeps?

Reply:
I have to revise this comment. The comment code here doesn't even work. I did have to add Atom[String] case, and I'm not sure why that should ever occur, but the example for reproducing it here doesn't work. There's at least one other place in XMLUtils where comments on XML wierdness about Text vs. PCData appear, and when I re-run those examples, they don't have the problems described in the comments anymore. I''ll fix the comment here, and update that other place to point out that the problem no longer occurs. Not sure what else I can do about this other than revisit all the use of XML parsers in Daffodil?